canonical-web-and-design/ubuntu.com
# Payment/renewal info parsing unit tests

## Done
- Added more tests around how we parse payment and renewal information that is then presented back to the user
- Light refactoring

## QA
- checkout this branch
- run `npm run test-js`
- see that the tests all pass

Fixes #7374
